build.gradle

```diff
@@ -46,12 +46,13 @@ dependencies {
     implementation 'com.azure:azure-identity:1.15.0'
 
     implementation 'com.azure.spring:azure-spring-boot-starter-active-directory:3.14.0'
-
     implementation 'com.h2database:h2'
+//    implementation 'org.postgresql:postgresql'
 
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
     testImplementation 'org.springframework.graphql:spring-graphql-test'
     testImplementation 'org.springframework.security:spring-security-test'
+//    testImplementation 'com.h2database:h2'
```

Comment:
Minor: Comments required?

Reply:
after we switch to Postgres, will need add this h2 back as test db.